Controller/Adminhtml/Item/MassDelete.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace SomethingDigital\AuctionItem\Controller\Adminhtml\Item;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Exception\NotFoundException;
+use Magento\Ui\Component\MassAction\Filter;
+use SomethingDigital\AuctionItem\Model\AuctionItemRepository;
+use SomethingDigital\AuctionItem\Model\ResourceModel\AuctionItem\CollectionFactory;
+
+/**
+ * Custom Controller for AuctionItem MassDelete
+ *
+ * Class MassDelete
+ */
+class MassDelete extends Action implements HttpPostActionInterface
+{
+    /**
+     * Authorization level
+     */
+    const ADMIN_RESOURCE = 'SomethingDigital_AuctionItem::grid';
```

Comment:
Looks like the const isn't used anywhere, can we get rid of it?

Reply:
I believe that the const overrides the ADMIN_RESOURCE in the parent which then gets used for the authorization.